engine/common/sys_con.c

```diff
@@ -248,21 +251,21 @@ void Sys_PrintLog( const char *pMsg )
 #ifdef XASH_COLORIZE_CONSOLE
 	Sys_PrintColorized( logtime, pMsg );
 #else
-	printf( "%s %s", logtime, pMsg );
+	printf( "%s%s", logtime, pMsg );
```

Comment:
Here you can replace printf by function similar to Sys_PrintColorized that will print but ignoring color codes. Except try to avoid copying, it's useless, and I don't know why PrintColorized was made that way.